nym-node/src/config/template.rs

```diff
@@ -185,6 +185,9 @@ announce_wss_port = {{#if entry_gateway.announce_wss_port }} {{ entry_gateway.an
 # derived shared keys, available client bandwidths and wireguard peers.
 clients_storage = '{{ entry_gateway.storage_paths.clients_storage }}'
 
+#Path to sqlite database containing all persistent stats data.
```

Comment:
nit: space before starting the text?

Reply:
Yes sir, fixed in https://github.com/nymtech/nym/pull/5022/commits/6486609040c2845c7785a029fad7c83f07ebf03a